orttraining/orttraining/training_ops/cuda/communication/send.cc

```diff
@@ -4,68 +4,116 @@
 #ifdef USE_HOROVOD
 
 #include "send.h"
+#include "common.h"
```

Comment:
let's have full path

Reply:
send, common or both?
This common.h is the local common we created before.